specification/network/resource-manager/Microsoft.Network/stable/2018-08-01/virtualNetwork.json

```diff
@@ -861,6 +865,49 @@
       ],
       "description": "ResourceNavigationLink resource."
     },
+    "ServiceDelegationPropertiesFormat": {
+      "properties": {
+        "serviceName": {
```

Comment:
Please check which of these properties are readonly and mark them accordingly

Reply:
These are not read-only